ConnectorRevit/RevitSharedResources/Models/ErrorEater.cs

```diff
@@ -40,7 +40,7 @@ public FailureProcessingResult PreprocessFailures(FailuresAccessor failuresAcces
         failuresAccessor.ResolveFailure(failure);
         resolvedFailures++;
       }
-      catch
+      catch (Autodesk.Revit.Exceptions.ApplicationException)
```

Comment:
We are logging Fatal, I guess is this Fatal for the plugin? How does the host app know the plugin is not healthy?

Reply:
I think we'd prefer to keep `Fatal` for cases where we are unsure if we've fully recovered (i.e. may have left DUI or the host app in an invalid state).

If this handler allows us to safely recover (i.e. we may have failed receiving, but we've properly rolled back), then I think I'd suggest `Error` level.
But we've historically been a little lossy goosy with Fatal vs Error.

Reply:
That being said.. I'm a bit puzzled why we're using a Task completion source here anyway. since we await the APIContext call anyway...

Reply:
I've lowered this log to be an error instead